catalog/security/catalog-security-xmlattributeplugin/src/main/java/org/codice/ddf/catalog/security/policy/xml/XmlAttributeSecurityPolicyPlugin.java

```diff
@@ -143,6 +143,11 @@ private void parseSecurityBlock(
         intersectionMap.get(name).add(valueSet);
       }
     }
+    try {
+      xmlStreamReader.close();
```

Comment:
❔ Did you need to explicitly call `close` here because XMLStreamReader does not implement closeable?

Reply:
I'm not entirely sure why XMLStreamReader does not use implement closeable but from what I can tell its own close() fits the same role. The Oracle docs describe it roughly the same as the close() for closable so I think it makes sense to make sure its closed.